test/functional/content_view/version/incremental_update_test.rb

```diff
@@ -26,7 +26,8 @@
 
     expect_foreman_task('3')
 
-    run_cmd(@cmd + params)
+    result = run_cmd(@cmd + params)
+    assert(result.exit_code, 0)
```

Comment:
@zjhuntin I hate to comment on an already-merged PR, but you want to use `assert_equals` here. Look at the [docs for `assert`](http://ruby-doc.org/stdlib-2.0.0/libdoc/minitest/rdoc/MiniTest/Assertions.html#method-i-assert).